src/views/count.vue

```diff
@@ -100,6 +100,12 @@
         product: "product/getCurrent"
       })
     },
+    mounted(){
+      this.fetchProduct(this.$route.params.sku)
+    },
```

Comment:
We don't need mounted hook here as we are using the ionViewWillEnter hook.

Reply:
Removed the mounted Hook.
Thank You sir for reviewing my code.